### ISSUE TITLE
Local sources:Cannot open dataset 

### ISSUE THREAD

**Issue by Evadzi** [CCI-Tools/cate#779]
### Expected behavior

*I expect cate 2.0.0. dev-20 to successfully concatenate and display GHG data(/neodc/esacci/ghg/data/obs4mips/crdp_3) downloaded from the esa ftp (anon-ftp.ceda.ac.uk ).*

### Actual behavior

*Cate 2.0.0. dev-20 is unable to open the local dataset and returns an error message*

### Copied error message

 *Traceback (most recent call last):
 ``` File "C:\Users\prosper\Miniconda3\envs\cate-env\lib\site-packages\cate\ds\local.py", line 183, in open_dataset
    monitor=monitor)
  File "C:\Users\prosper\Miniconda3\envs\cate-env\lib\site-packages\cate\core\ds.py", line 667, in open_xarray_dataset
    **kwargs)
  File "C:\Users\prosper\Miniconda3\envs\cate-env\lib\site-packages\xarray\backends\api.py", line 642, in open_mfdataset
    data_vars=data_vars, coords=coords)
  File "C:\Users\prosper\Miniconda3\envs\cate-env\lib\site-packages\xarray\core\combine.py", line 436, in auto_combine
    for ds in grouped]
  File "C:\Users\prosper\Miniconda3\envs\cate-env\lib\site-packages\xarray\core\combine.py", line 436, in <listcomp>
    for ds in grouped]
  File "C:\Users\prosper\Miniconda3\envs\cate-env\lib\site-packages\xarray\core\combine.py", line 365, in _auto_concat
    return concat(datasets, dim=dim, data_vars=data_vars, coords=coords)
  File "C:\Users\prosper\Miniconda3\envs\cate-env\lib\site-packages\xarray\core\combine.py", line 120, in concat
    return f(objs, dim, data_vars, coords, compat, positions)
  File "C:\Users\prosper\Miniconda3\envs\cate-env\lib\site-packages\xarray\core\combine.py", line 227, in _dataset_concat
    datasets = align(*datasets, join='outer', copy=False, exclude=[dim])
  File "C:\Users\prosper\Miniconda3\envs\cate-env\lib\site-packages\xarray\core\alignment.py", line 154, in align
    % (dim, sizes))
ValueError: arguments without labels along dimension 'sounding_dim' cannot be aligned because they have different dimension sizes: {6656, 7169, 4100, 5, 1542, 1543, 3592, 7177, 
5644, 6158, 8721, 7698, 3092, 2076, 5661, 6686, 4126, 7200, 5153, 5666, 2083, 5155, 3621, 7207, 3633, 5682, 6195, 4662, 3639, 6201, 7738, 5694, 8255, 7232, 7233, 6721, 4674, 3144, 
8266, 2637, 4173, 8271, 6736, 5199, 82, 4180, 4181, 6740, 5716, 6231, 4183, 6235, 2141, 7778, 2148, 3174, 616, 6248, 5226, 1645, 6765, 2159, 6256, 7280, 1139, 1140, 4724, 5112, 6264,
 2682, 7294, 5251, 9348, 4743, 2184, 7307, 2701, 7313, 7314, 4754, 6294, 6295, 4248, 7321, 3738, 5274, 2716, 3741, 5278, 5785, 3745, 5793, 4772, 7333, 2727, 1709, 6318, 5295, 4278, 6328, 
 5305, 186, 3262, 6847, 5831, 6857, 6859, 2767, 5839, 4303, 5331, 8404, 2779, 3804, 5341, 7167, 4835, 7396, 6372, 230, 6375, 6374, 4841, 2283, 6383, 5365, 8438, 6905, 4348, 6910, 3327,
 4865, 6914, 1283, 4866, 4867, 7435, 5390, 2831, 6926, 5905, 4369, 4371, 2323, 7446, 7960, 4895, 6947, 5931, 4398, 816, 819, 8500, 5941, 6453, 5943, 5429, 7481, 5437, 7998, 6464, 4418, 
 6467, 6466, 2374, 5959, 4938, 5451, 7501, 5967, 6481, 4433, 5460, 343, 7514, 7005, 7518, 5983, 6496, 4960, 5986, 4963, 2920, 7016, 5480, 7531, 7022, 7536, 7539, 8563, 9079, 5495, 889, 
 5498, 3963, 5499, 4480, 4993, 6025, 2953, 7565, 5518, 6030, 2961, 6546, 5013, 5525, 6042, 6555, 3996, 4509, 5022, 5023, 5024, 9121, 5539, 4516, 6052, 5542, 2985, 3499, 6573, 5550, 5039, 
 7088, 4527, 4530, 7091, 8118, 5562, 6075, 3517, 7102, 4031, 8127, 7103, 6084, 5063, 1992, 5579, 4556, 6093, 7121, 8658, 5077, 7127, 5080, 2009, 4057, 4060, 5597, 6109, 6623, 7135, 3553, 
 7650, 7139, 5602, 4576, 3039, 2536, 5609, 3560, 4080, 8177, 2034, 7155, 7153, 5621, 7152, 7665, 7672, 6643, 5118, 6143}

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "C:\Users\prosper\Miniconda3\envs\cate-env\lib\site-packages\cate\util\web\jsonrpchandler.py", line 209, in send_service_method_result
    result = future.result()
  File "C:\Users\prosper\Miniconda3\envs\cate-env\lib\concurrent\futures\_base.py", line 425, in result
    return self.__get_result()
  File "C:\Users\prosper\Miniconda3\envs\cate-env\lib\concurrent\futures\_base.py", line 384, in __get_result
    raise self._exception
  File "C:\Users\prosper\Miniconda3\envs\cate-env\lib\concurrent\futures\thread.py", line 56, in run
    result = self.fn(*self.args, **self.kwargs)
  File "C:\Users\prosper\Miniconda3\envs\cate-env\lib\site-packages\cate\util\web\jsonrpchandler.py", line 306, in call_service_method
    result = method(*method_params, monitor=monitor)
  File "C:\Users\prosper\Miniconda3\envs\cate-env\lib\site-packages\cate\webapi\websocket.py", line 292, in set_workspace_resource
    monitor=monitor)
  File "C:\Users\prosper\Miniconda3\envs\cate-env\lib\site-packages\cate\core\wsmanag.py", line 320, in set_workspace_resource
    workspace.execute_workflow(res_name=res_name, monitor=monitor)
  File "C:\Users\prosper\Miniconda3\envs\cate-env\lib\site-packages\cate\core\workspace.py", line 662, in execute_workflow
    self.workflow.invoke_steps(steps, context=self._new_context(), monitor=monitor)
  File "C:\Users\prosper\Miniconda3\envs\cate-env\lib\site-packages\cate\core\workflow.py", line 627, in invoke_steps
    steps[0].invoke(context=context, monitor=monitor)
  File "C:\Users\prosper\Miniconda3\envs\cate-env\lib\site-packages\cate\core\workflow.py", line 318, in invoke
    self._invoke_impl(_new_context(context, step=self), monitor=monitor)
  File "C:\Users\prosper\Miniconda3\envs\cate-env\lib\site-packages\cate\core\workflow.py", line 980, in _invoke_impl
    return_value = self._op(monitor=monitor, **input_values)
  File "C:\Users\prosper\Miniconda3\envs\cate-env\lib\site-packages\cate\core\op.py", line 216, in __call__
    return_value = self._wrapped_op(**input_values)
  File "C:\Users\prosper\Miniconda3\envs\cate-env\lib\site-packages\cate\ops\io.py", line 83, in open_dataset
    monitor=monitor)
  File "C:\Users\prosper\Miniconda3\envs\cate-env\lib\site-packages\cate\core\ds.py", line 591, in open_dataset
    return data_source.open_dataset(time_range, region, var_names, monitor=monitor)
  File "C:\Users\prosper\Miniconda3\envs\cate-env\lib\site-packages\cate\ds\local.py", line 197, in open_dataset
    raise ValidationError(msg) from e
cate.core.types.ValidationError: Cannot open local dataset:
arguments without labels along dimension 'sounding_dim' cannot be aligned because they have different dimension sizes: {6656, 7169, 4100, 5, 1542, 1543, 3592, 7177, 5644, 6158, 8721, 7698,
 3092, 2076, 5661, 6686, 4126, 7200, 5153, 5666, 2083, 5155, 3621, 7207, 3633, 5682, 6195, 4662, 3639, 6201, 7738, 5694, 8255, 7232, 7233, 6721, 4674, 3144, 8266, 2637, 4173, 8271, 6736, 5199,
 82, 4180, 4181, 6740, 5716, 6231, 4183, 6235, 2141, 7778, 2148, 3174, 616, 6248, 5226, 1645, 6765, 2159, 6256, 7280, 1139, 1140, 4724, 5112, 6264, 2682, 7294, 5251, 9348, 4743, 2184, 7307,
 2701, 7313, 7314, 4754, 6294, 6295, 4248, 7321, 3738, 5274, 2716, 3741, 5278, 5785, 3745, 5793, 4772, 7333, 2727, 1709, 6318, 5295, 4278, 6328, 5305, 186, 3262, 6847, 5831, 6857, 6859, 2767,
 5839, 4303, 5331, 8404, 2779, 3804, 5341, 7167, 4835, 7396, 6372, 230, 6375, 6374, 4841, 2283, 6383, 5365, 8438, 6905, 4348, 6910, 3327, 4865, 6914, 1283, 4866, 4867, 7435, 5390, 2831, 6926,
 5905, 4369, 4371, 2323, 7446, 7960, 4895, 6947, 5931, 4398, 816, 819, 8500, 5941, 6453, 5943, 5429, 7481, 5437, 7998, 6464, 4418, 6467, 6466, 2374, 5959, 4938, 5451, 7501, 5967, 6481, 4433, 
 5460, 343, 7514, 7005, 7518, 5983, 6496, 4960, 5986, 4963, 2920, 7016, 5480, 7531, 7022, 7536, 7539, 8563, 9079, 5495, 889, 5498, 3963, 5499, 4480, 4993, 6025, 2953, 7565, 5518, 6030, 2961, 6546,
 5013, 5525, 6042, 6555, 3996, 4509, 5022, 5023, 5024, 9121, 5539, 4516, 6052, 5542, 2985, 3499, 6573, 5550, 5039, 7088, 4527, 4530, 7091, 8118, 5562, 6075, 3517, 7102, 4031, 8127, 7103, 6084,
 5063, 1992, 5579, 4556, 6093, 7121, 8658, 5077, 7127, 5080, 2009, 4057, 4060, 5597, 6109, 6623, 7135, 3553, 7650, 7139, 5602, 4576, 3039, 2536, 5609, 3560, 4080, 8177, 2034, 7155, 7153, 5621, 
 7152, 7665, 7672, 6643, 5118, 6143}* ```


### Specifications 

*cate 2.0.0. dev-20, Windows 10*

**Comment by forman** [comment]
This is a bug, because Cate should be able to open all NetCDF datasets.

**Comment by Evadzi** [comment]
Issue resolved in cate 2.0.0-dev 22. I tried for ghg and fire data..
